packages/app/src/ui/components/modal-dialog/index.ts

```diff
@@ -6,3 +6,4 @@
 export * from "./modal-dialog.component"
 export * from "./modal-dialog.interface"
 export * from "./modal-dialog.styles"
+export { getModalDialogStyle } from "App/ui/components/modal-dialog/get-modal-dialog-style.helper"
```

Comment:
Seems easier + doesn't need to be modified when changing  `get-modal-dialog-style.helper`
```suggestion
export * from "./get-modal-dialog-style.helper"
```

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/b1e89e36ec965f086ebdbf036cf35ef7965c500d